features/gba-dev/gezagsverhouding-gba.feature

```diff
@@ -0,0 +1,142 @@
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'gezagsverhouding' gegevens
+    | naam                                 | waarde |
+    | indicatie gezag minderjarige (32.10) | 12     |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000152                       |
+    | fields              | <naam>                          |
+    Dan heeft de response een persoon met de volgende gegevens
+    | naam                                    | waarde           |
+    | indicatieGezagMinderjarige.code         | 12               |
+    | indicatieGezagMinderjarige.omschrijving | ouder1 en ouder2 |
+
+    Voorbeelden:
+    | naam                                    |
+    | indicatieGezagMinderjarige              |
+    | indicatieGezagMinderjarige.code         |
+    | indicatieGezagMinderjarige.omschrijving |
+
+  Scenario: persoon heeft 'gezagsverhouding' veld: 'indicatie curateleregister (33.10)'
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'gezagsverhouding' gegevens
+    | naam                               | waarde |
+    | indicatie curateleregister (33.10) | 1      |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000152                       |
+    | fields              | indicatieCurateleRegister       |
+    Dan heeft de response een persoon met de volgende gegevens
+    | naam                      | waarde |
+    | indicatieCurateleRegister | true   |
+
+  Abstract Scenario: 'gezagsverhouding' veld(en) van persoon is/zijn in onderzoek
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'gezagsverhouding' gegevens
+    | naam                            | waarde                    |
+    | aanduiding in onderzoek (83.10) | <gba in onderzoek waarde> |
+    | datum ingang onderzoek (83.20)  | 20020701                  |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000152                       |
+    | fields              | inOnderzoek                     |
+    Dan heeft de response een persoon met de volgende gegevens
+    | naam                                              | waarde                            |
+    | gezagInOnderzoek.aanduidingGegevensInOnderzoek    | <gba in onderzoek waarde>         |
+    | gezagInOnderzoek.datumIngangOnderzoek             | 20020701                             |
+
+
+    Voorbeelden:
+    | gba in onderzoek waarde | gezag minderjarige in onderzoek | curateleregister in onderzoek |
```

Comment:
De kolommen 'gezag minderjarige in onderzoek' en 'curatele in onderzoek' worden niet gebruikt. Kan dus weg

Reply:
Aangepast